docs: add note about nigthly builds repository (#6025)
eclipse/kura: docs/getting-started/install-kura.md

!!! note
    The above instructions might require `sudo` privileges depending on your system configuration.

??? tip "Nightly builds APT repository"

>     For users who want to test the latest development builds of Eclipse Kura&trade;, a development APT repository is available. This repository contains pre-release versions and should only be used for testing purposes.

Reviewer: ```suggestion
    For users who want to test the latest development builds of Eclipse Kura&trade;, a development APT repository is available. This repository includes pre-release versions and is intended solely for testing purposes.
```

Possible alternative

Author: Better than my version
